11.curiosity.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #39.            pred_s_next = F.relu(self.pred_module1( torch.cat([feature_x, a_vec.float()], dim = -1).detach()))
</pre><p>suggestion (Black Style):</p><p> </p><pre class="ql-syntax" spellcheck="false">pred_s_next = F.relu(
    self.pred_module1(torch.cat([feature_x, a_vec.float()], dim =-1).detach())
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Curt-Park/rainbow-is-all-you-need/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>